overlay.d/07el9/usr/lib/systemd/system-preset/36-iscsid-disabled.preset

```diff
@@ -1 +1,7 @@
 disable iscsid.socket
```

Comment:
Aside: this came in via https://github.com/openshift/os/commit/929ac48d327cdf414ea0bfa603e252174be8a0d5 (in https://github.com/openshift/os/pull/905)which doesn't have any context on what exactly this fixes, which isn't great. @travier, do you know why we needed this? I'd like to add a comment there (and otherwise just drop it).

Reply:
Arg sorry, it was meant as a workaround to be investigated later and then I forgot.

Reply:
I think I disabled it as it was an additionally enabled unit and that made our kola test fail.

Reply:
Ack. I'll remove it in a follow-up PR and if anything breaks, at least we'll be able to add some context to it. :)

Reply:
Sorry, I hadn't seen your second comment here. If that's the case, I think that was fixed by https://github.com/coreos/coreos-assembler/pull/3275.

Reply:
So we did "fix" it but we stopped checking for the "iscsid.service" being disabled at the same time thus this issue.

Reply:
My bad, we disable the `iscsi.service` service here, not the `d` one.

Reply:
We can drop this here or in another PR. Up to you.

Reply:
Yeah, I'll drop it in a follow-up.

Reply:
Opened follow-up in https://github.com/openshift/os/pull/1298.